src/renderers/shaders/ShaderLib/meshphong_vert.glsl

```diff
@@ -40,9 +40,9 @@ void main() {
 #endif
 
 	#include <begin_vertex>
-	#include <displacementmap_vertex>
 	#include <morphtarget_vertex>
 	#include <skinning_vertex>
+	#include <displacementmap_vertex>
```

Comment:
Since displacement is done along the post-transformed normal, displacement mapping needs to take place after position vertices have been transformed.